physionet-django/project/modelcomponents/activeproject.py

```diff
@@ -80,7 +84,11 @@ class ActiveProject(Metadata, UnpublishedProject, SubmissionInfo):
     MAX_SUBMITTING_PROJECTS = 10
     INDIVIDUAL_FILE_SIZE_LIMIT = 10 * 1024**3
     # Where all the active project files are kept
-    FILE_ROOT = os.path.join(settings.MEDIA_ROOT, 'active-projects')
+
+    FILE_ROOT = os.path.join(
+        (settings.MEDIA_ROOT if settings.STORAGE_TYPE == StorageTypes.LOCAL else settings.GCP_STORAGE_BUCKET_NAME),
```

Comment:
Maybe we can add a property like `FILE_ROOT` to the ProjectFile classes to get rid of this